src/streamlit_extras/annotated_text/__init__.py

```diff
@@ -1,5 +1,9 @@
 from annotated_text import annotated_text
 
+from .. import extra
+
+annotated_text = extra(annotated_text)
```

Comment:
💡  In case the extra comes from an other library, just define it using the function (and not the decorator)